### PR TITLE
Add unpack_collection_qa tool.

### PR DESCRIPTION
**Pull request by raydittmeier**
L-LDOPE tools update to add the collection_qa tool

### CODE REVIEW

File: src/unpack_collection_bits.c [hunk 1]
```diff
@@ -0,0 +1,1315 @@
...
+8/31/2016     Ray Dittmeier    Original Development
+
+NOTES:
+******************************************************************************/
+short unpack_bits
+(
+    char *qa_infile,      /* I: input QA filename */
+    char *qa_outfile,     /* I: output QA base filename */
+    bool qa_specd[NQUALITY_TYPES],  /* I: array to specify which QA bands
+                                          was specified for processing */
```
Comment (gschmidt-usgs) [inline src/unpack_collection_bits.c]: Change 'was' to 'were'

File: src/unpack_collection_qa.c [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+Date        Programmer       Reason
+--------    ---------------  -------------------------------------
+8/31/2016     Ray Dittmeier    Original Development
+
+NOTES:
+******************************************************************************/
+void usage ()
+{
+    printf ("unpack_collection_qa will read the QA band from the Landsat L4-8 "
+            "input file, then unpack this band into individual QA bands "
```
Comment (gschmidt-usgs) [inline src/unpack_collection_qa.c]: Should probably explicitly state "collection" in the help information, even though it's in the application name.  
Suggest changing "input file" to "input Collection product"

**Comment by gschmidt-usgs** [comment]
Checked the bits to make sure they matched the documentation, including whether they were single or double bit values.  Everything in the code lines up with the documentation.  No further comments.

File: src/unpack_collection_bits.c (hunk 1 above)
Comment (raydittmeier) [inline src/unpack_collection_bits.c]: Changed to 'were'  (in 2 spots)

File: src/unpack_collection_qa.c (hunk 2 above)
Comment (raydittmeier) [inline src/unpack_collection_qa.c]: Changed help text to use "input Collection product"

**Pull request merged**